### DISCUSSION TITLE
will spdlog add filter function

### DISCUSSION THREAD

**Discussion by jiudianren** [gabime/spdlog#2490] [Ideas]
i see an issue look like my question at [](https://github.com/gabime/spdlog/issues/1678).

i want a filter option , like boost log , like this:

![image](https://user-images.githubusercontent.com/9455812/191007656-d1a13fdd-0d53-4c72-a8a1-517d126e97bc.png)

i want to add more attributes, and later ,i can filter some of them to some of my sinks.

i want to ask that will spdlog supptor this function like boost log .
thanks for your answer.

**Comment by tt4g** [comment]
spdlog does not support custom attributes (#532, #617).
Therefore, it cannot provide a filter like Boost.Log.
